src/server/game/Entities/Player/SocialMgr.cpp

```diff
@@ -309,15 +307,15 @@ void SocialMgr::BroadcastToFriendListers(Player* player, WorldPacket* packet)
     }
 }
 
-PlayerSocial* SocialMgr::LoadFromDB(PreparedQueryResult result, uint32 guid)
+PlayerSocial* SocialMgr::LoadFromDB(PreparedQueryResult result, uint64 guid)
 {
     PlayerSocial* social = &m_socialMap[guid];
     social->SetPlayerGUID(guid);
 
     if (!result)
         return social;
 
-    uint32 friendGuid = 0;
+    uint64 friendGuid = 0;
```

Comment:
can't we declare `friendGuid` inside the `do` loop?

```c++
auto friendGuid = fields[0].GetUInt32();
```

Reply:
auto is unsafe shit but okay